src/OrchardCore.Modules/OrchardCore.Resources/Assets/trumbowyg/js/trumbowyg.js

```diff
@@ -461,7 +461,7 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
             'i': 'em',
             's': 'del',
             'strike': 'del',
-            'div': 'p'
+            'div': 'div'
```

Comment:
Why not just delete this line? Why translate `div` to `div`?

Reply:
Because I believe this is the recommended method of overriding tag conversions based on their documentation:
[https://alex-d.github.io/Trumbowyg/documentation/#semantic](https://alex-d.github.io/Trumbowyg/documentation/#semantic)
> Since 2.12.0 you can also put an object to customize the semantic tag mapping for each one of these tags: b, i, s, strike, div. 

`$('.trumbowyg').trumbowyg({
    semantic: {
        'div': 'div' // Editor does nothing on div tags now
    }
});`

Reply:
Ah got it thanks. I didn't realize that is a fixed set of tags with `'div': 'p'` as default.